src/graphql/queries/ListArticles.js

```diff
@@ -443,6 +454,35 @@ export default {
       });
     }
 
+    // FIXME: Remove else statement after implementing media article on rumor-site
+    if (filter.articleTypes) {
+      filterQueries.push({
+        terms: {
+          articleType: filter.articleTypes,
+        },
+      });
+    } else if (!filter.mediaUrl) {
+      filterQueries.push({
+        term: {
+          articleType: 'TEXT',
```

Comment:
Question: should we migrate all existing articles to have `articleType` = `TEXT` after deploying it?
Seems that a simple curl command can do this.

Reply:
Updated in PR description